pkg/pdc/client.go

```diff
@@ -32,12 +32,14 @@ var (
 type Config struct {
 	Token           string
 	HostedGrafanaID string
+	Network         string
 	URL             *url.URL
 }
 
 func (cfg *Config) RegisterFlags(fs *flag.FlagSet) {
 	fs.StringVar(&cfg.Token, "token", "", "The token to use to authenticate with Grafana Cloud. It must have the pdc-signing:write scope")
 	fs.StringVar(&cfg.HostedGrafanaID, "gcloud-hosted-grafana-id", "", "The ID of the Hosted Grafana instance to connect to")
+	fs.StringVar(&cfg.Network, "network", "", "The ID of the PDC network to connect to")
```

Comment:
Just a small nit - should this be referenced as `NetworkID` instead of just `Network`? I think it makes it clearer what this var is to be specific

Reply:
It could be. I don't really feel like it is an id because it can be anything 🤔 What do you think?

Reply:
We need to avoid name collisions when looking up the SSH connection pool to use for each network. We can use a combination of network name and stackId for the SSH connection pool lookup to avoid the need for globally unique network names but we would still have to enforce them to be unique per stack. Maybe that is something the plugin can do at creation time and is a non issue. 

That is a long way of saying I suppose we can use name instead of id for the network here and just refer to it as "network" but can we update the description to say "The name of the PDC network..." to be more accurate?

Reply:
Thanks, LGTM